crypto/asn1/a_time.c

```diff
@@ -147,10 +147,9 @@ ASN1_GENERALIZEDTIME *ASN1_TIME_to_generalizedtime(ASN1_TIME *t,
     if (!out || !*out) {
         if (!(ret = ASN1_GENERALIZEDTIME_new()))
             goto err;
-        if (out)
-            *out = ret;
-    } else
+    } else {
         ret = *out;
+    }
```

Comment:
@richsalz actually that bit *is* documented. The style guide says: "If one of the branches is a compound statement, then use braces on both parts"